frontend.tf

```diff
@@ -21,30 +21,11 @@ variable "grr_frontend_image_tag" {
   description = "Docker image tag to pull of image specified by grr_frontend_image"
 }
 
-variable "grr_frontend_port" {
-  description = "GRR frontend port that clients will connect to"
-  default     = 443
-}
-
-variable "grr_frontend_monitoring_port" {
-  description = "GRR frontend monitoring stats port"
-  default     = 5222
-}
-
-variable "grr_frontend_network_tag" {
-  description = "Firewall network tag to open ports for GRR frontend"
-  default     = "grr-frontend"
-}
-
 variable "grr_frontend_target_size" {
```

Comment:
If we want to move to the standard module format, I think all variable declarations should be moved to `variables.tf`, or none at all. If you think we should use `variables.tf`, please move all variable declarations there.

Reply:
Sounds good! I've moved all the variables declarations to variables.tf